cli/cobra.go

```diff
@@ -59,7 +59,10 @@ func Client(apiUlr, userID, password, otp string) (c *proxmox.Client, err error)
 	if otp == "" {
 		otp = os.Getenv("PM_OTP")
 	}
-	c, err = proxmox.NewClient(apiUlr, nil, tlsconf, proxyUrl, timeout)
+	if http_headers == "" {
+		http_headers = os.Getenv("HTTP_HEADERS")
```

Comment:
I would use PM_HTTP_HEADERS for coherence with other envs, what do you think?

Reply:
Works out for me.